src/D2L.Bmx/Okta/OktaApi.cs

```diff
@@ -95,41 +102,55 @@ string challengeResponse
 		var request = new VerifyMfaChallengeResponseRequest(
 			StateToken: stateToken,
 			PassCode: challengeResponse );
-
-		var resp = await _httpClient.PostAsJsonAsync(
-			$"authn/factors/{factorId}/verify",
-			request,
-			SourceGenerationContext.Default.VerifyMfaChallengeResponseRequest );
-
+		HttpResponseMessage resp;
+		try {
+			resp = await _httpClient.PostAsJsonAsync(
+				$"authn/factors/{factorId}/verify",
+				request,
+				SourceGenerationContext.Default.VerifyMfaChallengeResponseRequest );
+		} catch( Exception ex ) {
+			throw new BmxException( "Error verifying Okta MFA challenge response. Try checking for a stable connection.", ex );
```

Comment:
similarly here (and below). the message should be reworded to cover broader possible causes